src/main/java/com/nowoczesnyjunior/financialapp/expensemodule/service/ExpenseServiceImpl.java

```diff
@@ -0,0 +1,111 @@
+package com.nowoczesnyjunior.financialapp.expensemodule.service;
+
+import com.nowoczesnyjunior.financialapp.expensemodule.exception.CategoryNotFoundException;
+import com.nowoczesnyjunior.financialapp.expensemodule.exception.InvalidDateException;
+import com.nowoczesnyjunior.financialapp.expensemodule.mapper.ExpenseMapper;
+import com.nowoczesnyjunior.financialapp.expensemodule.model.Expense;
+import com.nowoczesnyjunior.financialapp.expensemodule.repository.CategoryRepository;
+import com.nowoczesnyjunior.financialapp.expensemodule.repository.ExpenseRepository;
+import com.nowoczesnyjunior.financialapp.openapi.model.CategoryDto;
+import com.nowoczesnyjunior.financialapp.openapi.model.ExpenseDto;
+import com.nowoczesnyjunior.financialapp.usermodule.model.User;
+import com.nowoczesnyjunior.financialapp.usermodule.repository.UserRepository;
+import org.hibernate.ObjectNotFoundException;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class ExpenseServiceImpl implements ExpenseService {
+
+    private final ExpenseRepository expenseRepository;
+    private final CategoryRepository categoryRepository;
+    private final UserRepository userRepository;
+    private final ExpenseMapper expenseMapper;
+
+    LocalDateTime defaultStartDate = LocalDateTime.of(1970, 01, 01, 00, 00);
+    LocalDateTime defaultEndDate = LocalDate.now().atStartOfDay();
+
+    public ExpenseServiceImpl(ExpenseRepository expenseRepository, CategoryRepository categoryRepository, UserRepository userRepository, ExpenseMapper expenseMapper) {
```

Comment:
You are using Lombok in other places, to stay consistent - how about using `@RequiredArgsConstructor`?

Reply:
done